pygotham/frontend/talks.py

```diff
@@ -100,6 +100,13 @@ def proposal(pk=None):
         db.session.add(talk)
         db.session.commit()
 
+        speaker = Speaker(
```

Comment:
I am not sure we should have a concept of the 'primary speaker'. What would that afford us? Also, what beyond 'user_id' & 'recording_release' would we want to drop from the talk model?